python/mangadap/proc/templatelibrary.py

```diff
@@ -1067,6 +1071,20 @@ def _process_library(self, wavelength_range=None, renormalize=True):
                                                        regular=self.library['in_vacuum'])
                                         for w in self.hdu['WAVE'].data])
 
+        # TODO: I think this kludge was necessary because of a bug in
+        # angstroms_per_pixel.  If regular was True and log was False,
+        # it only returns a single integer, otherwise it returned a
+        # vector.  I've changed angstroms_per_pixel, but have left the
+        # kludge below for now for testing.
+
```

Comment:
See the note above.  I put a warning in the code in case the kludge is triggered.  I think we'll keep it around for a while and remove it when we're confident it's not being triggered.